features/step_definitions/appointment_summary_steps.rb

```diff
@@ -0,0 +1,35 @@
+Given(/^I am logged in as a Pension Wise Administrator$/) do
+  @administrator = create(:user, admin: true).tap(&:confirm!)
+
+  page = UserSignInPage.new
+  page.login(@administrator)
+end
+
+Given(/^there are existing Appointment Summaries$/) do
+  create_list(
+    :appointment_summary,
+    Kaminari.config.default_per_page + 1
+  )
+end
+
+When(/^I visit the Summary Browser$/) do
+  @page = AppointmentSummaryBrowserPage.new
+  @page.load
+end
+
+Then(/^I am presented with Appointment Summaries$/) do
+  expect(@page).to have_appointments(count: Kaminari.config.default_per_page)
+end
+
+Then(/^I see there are multiple pages$/) do
+  expect(@page).to have_pages
+end
+
+Then(/^the date range defaults to the last month$/) do
+  expect(@page.start_date.value).to be_present
+  expect(@page.end_date.value).to be_present
```

Comment:
I feel this is misleading to anyone reading the feature as the step definition does not match the implementation.

Feel like changing the step name here is the best approach rather than the implementation.